Gaijinizer patch/Map537.txt

```diff
@@ -4,243 +4,310 @@
 
 ボクが管轄してた部分も、ここから見える範囲ですら
 砂の一粒にも満たないんだろうなぁ……
-> CONTEXT: Map537/events/7/pages/0/7/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/7/Dialogue/enemy29
+The multiverse...
 
+My jurisdiction along with everything I can see
+from here is nothing but a speck...#Ron
 > END STRING
 
 > BEGIN STRING
 同じ正史の並行世界ですら、無限に近いほどあるのに
 
 それがこうやって束ねられて、星に形容されて……
 視覚情報として出されると、これまた途方もない話だね…
-> CONTEXT: Map537/events/7/pages/0/12/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/12/Dialogue/enemy29
+Even though the parallel worlds of the Official
+timeline are bundled together like this, in the
+shape of stars and visually displayed, it's
+a pretty crazy thing...#RonIffy
 > END STRING
 
 > BEGIN STRING
 『並行世界』といえば………
 ルカくんの世界は、もう大詰めなんだね……
 
 ボクはてっきり、まだ先の事かと思ってたよ……
-> CONTEXT: Map537/events/7/pages/0/18/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/18/Dialogue/enemy29
+Though it's a parallel world, Luka-kun's world is
+already at its finale. I thought for sure it had
+more time before then...#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 『第３の選択』……
 
 事態を収拾するためとはいえ、えげつないよね…
 ひとつの特異点に背負わせる業としては……
-> CONTEXT: Map537/events/7/pages/0/23/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map537/events/7/pages/0/23/Dialogue/enemy29
+"A third option"...
+Even if it's to resolve the situation... it's
+disgusting that he's being burdened with the role
+of singularity.#Ron
 > END STRING
 
 > BEGIN STRING
 ボクには無関係な世界だから、といえばそこまでだけど
 
 辛いだろうなぁ……ルカくん。
-> CONTEXT: Map537/events/7/pages/0/28/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/28/Dialogue/enemy29
+That's all I can say since that world has nothing
+to do with me.
 
+But it must be rough, Luka-kun...#Ron
 > END STRING
 
 > BEGIN STRING
 ………！！
 この感じ……\\C[13]『白兎』\\C[0]…！？
 
 い、いや、そんなわけない……！！
-> CONTEXT: Map537/events/7/pages/0/53/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map537/events/7/pages/0/53/Dialogue/enemy29
+......!!
 
+That sensation... \\C[13]White Rabbit\\C[0]?!
+
+N-No, it can't be!!#Ron
 > END STRING
```

Comment:
Did you put this in two text boxes on purpose?

Reply:
Yeah, mostly for emphasis. Feel free to change, though.

Reply:
Nope, just wanted to make sure. It's fine by me. 👍